Misc/reputations.json

```diff
@@ -98,26 +100,13 @@
       "regex": "\\b[a-fA-F\\d]{40}\\b",
       "reputationCommand": "file",
       "details": "File SHA1",
```

Comment:
please change to `SHA-1`

Reply:
Are you sure? It seems the convention has been without the hyphen so I don't think we should change them (and if we should, what about the SHA1/256 in the `toVersion: "3.0.1"` parts?
![image](https://user-images.githubusercontent.com/43602124/52180812-7aa9df00-27f3-11e9-8b55-423a5fad35a8.png)

Reply:
We are changing this convention according to the standard set by @kirbles19 
All SHA indicators should be `SHA-...`

Reply:
Not in the context, in the details (only UI)

Reply:
I meant in `details`